### PR TITLE
Fix: send back error

### PR DESCRIPTION
**Pull request by alexandre-merle**
Update ready-set-stream npm version, and add
non-regression testing.

Fixes https://github.com/scality/S3/issues/334

### CODE REVIEW

**Comment by alexandre-merle** [comment]
Test for non-regression will come soon

**Comment by alexandre-merle** [comment]
non regression test added

**Comment by alexandre-merle** [comment]
@ironman-machine let's do it

**Comment by alexandre-merle** [comment]
@ironman-machine again

File: tests/unit/utils/responseStreamData.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import assert from 'assert';
 import httpMocks from 'node-mocks-http';
 import { EventEmitter } from 'events';
+import { errors } from 'arsenal';
```
Comment (GuillaumeGomez) [inline tests/unit/utils/responseStreamData.js]: Why the "{}" around errors?

**Comment by alexandre-merle** [comment]
@ironman-machine let's do it

**Comment by alexandre-merle** [comment]
@ironman-machine again

**Pull request merged**